components/core/CMakeLists.txt

```diff
@@ -130,7 +130,13 @@ else()
 endif()
 
 # Find and setup msgpack
-find_package(msgpackc-cxx 5.0.0 REQUIRED)
+if(APPLE)
+    execute_process(COMMAND brew --prefix msgpack-cxx OUTPUT_VARIABLE PREFIX_MSGPACK_CXX)
+    string(STRIP "${PREFIX_MSGPACK_CXX}" PREFIX_MSGPACK_CXX)
+endif()
+find_package(msgpackc-cxx 5.0.0 REQUIRED
+        HINTS ${PREFIX_MSGPACK_CXX}
```

Comment:
For other systems, `PREFIX_MSGPACK_CXX` would just be blank, which will not harm the search process.